app/components/Tabs.vue

```diff
@@ -31,6 +31,7 @@
   width: 100%;
   padding: 0 15px;
   height: 54px;
+  z-index: 10;
```

Comment:
@gettinToasty recently rid of z-index here, what is the reason to add it now?

Reply:
prolly just a merge conflict regression